cmd/assets/list.go

```diff
@@ -63,8 +63,8 @@ var listAssetsCmd = &cobra.Command{
 }
 
 func init() {
-	rootCmd.AddCommand(listAssetsCmd)
+	Cmd.AddCommand(listAssetsCmd)
```

Comment:
prints full object vs list of assets. i would expect same behaviour as list products

Reply:
Fixed